### ISSUE TITLE
Unmet peer dependency - Webpack 4

### ISSUE THREAD

**Issue by bartholomej** [rubenspgcavalcante/webpack-chrome-extension-reloader#34]
**Type:**
- [ ] bug  
- [ ] feature
- [x] enhancement
- [ ] question
 
**Environment:**
 - OS: All
 - Version: 0.6.5 
 
**Going to open a PR:**
- [x] yes
- [ ] no
 
**Description:**  
``` 
webpack-chrome-extension-reloader@0.6.5" has incorrect peer dependency "webpack@2 | 3".
```
Webpack 4 is stable and your great extension works like a charm with current version 4.1.0. I am using it here: https://github.com/bartholomej/csfd-magnets
I'll make PR for updating peer dependencies.

**Comment by RShergold** [comment]
Came here to ask for this exact thing :smile:

**Comment by rubenspgcavalcante** [comment]
Published on latest version 0.6.6 🎉
